core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -42,6 +42,7 @@ import org.apache.spark.deploy.history.config._
 import org.apache.spark.internal.Logging
 import org.apache.spark.scheduler._
 import org.apache.spark.scheduler.ReplayListenerBus._
+import org.apache.spark.status.KVUtils._
```

Comment:
I start using more stuff from that object in later changes, so might as well avoid these changes later.

In fact I'm not sure why I'm not just using `open()` in this class now, since it was added to `KVUtils`...